backbone.js

```diff
@@ -867,7 +867,7 @@
         for (var i = 0; i < this.length; i++) {
           if (!modelMap[(model = this.models[i]).cid]) toRemove.push(model);
         }
-        if (toRemove.length) this._removeModels(toRemove, options);
+        if (toRemove.length) previousModels = this._removeModels(toRemove, options);
```

Comment:
Everything in `toRemove` has already been [verified](https://github.com/jashkenas/backbone/pull/3711/files#diff-0d56d0d310de7ff18b3cef9c2f8f75dcR868) from the current models, shouldn't need to grab `#_removeModels` return.